src/components/data_comps/datm/datm_comp_mod.F90

```diff
@@ -312,9 +312,6 @@ subroutine datm_comp_init(Eclock, x2a, a2x, &
        k = mct_aVect_indexRA(SDATM%grid%data,'frac')
        SDATM%grid%data%rAttr(k,:) = 1.0_R8
 
-       !--- set data needed for cosz t-interp method ---
-       call shr_strdata_setOrbs(SDATM,orbEccen,orbMvelpp,orbLambm0,orbObliqr,idt)
```

Comment:
Note that I removed the setOrbs call from datm_comp_init. There is now a setOrbs call in datm_comp_run, and datm_comp_run is called from near the end of datm_comp_init, so this seemed like a safe thing to do. But I wasn't 100% confident in this being safe, which was part of my motivation for running a handful of tests to confirm I hadn't changed answers.

If others feel this removal is potentially problematic, I can put it back in.